frontend/lib/uast/post-parse-checks.cpp

```diff
@@ -262,13 +262,18 @@ bool Visitor::isParentFalseBlock(int depth) const {
   return false;
 }
 
-void Visitor::checkForOneElementArraysWithoutComma(const Array* node) {
-  if (!node->hasTrailingComma() && node->numChildren() == 1) {
-    warn(node, "single-element array literals without a trailing comma are "
-         "deprecated; please rewrite as '%s'",
-         node->isAssociative() ? "[myKey => myElem, ]" : "[myElem, ]");
+void Visitor::checkForArraysOfRanges(const Array* node) {
```

Comment:
This function name is plural and the enum name is singular

Reply:
I made the enum name singular to match the user's flag.  I'd already written this routine before introducing the flag, though, and thought of it as checking for all such cases which is why it uses the plural.  Scanning other visitor checks, though, I see that they often use singular names even though they're also checking for all cases, such as "checkPrivateDecl" rather than "checkPrivateDecls".  Let me know if you'd prefer for this to change to something else ("checkForOneElementArraysOfRangeLiterals" is probably the most accurate description, but felt long).

Reply:
I just put a comment here to mark it as I mentioned in the approval comment that there were a couple of places where I noticed the difference. I'm fine with this being the name, just wanted to make it easy to find the parts I was calling attention to.